test/suites/greenboot.robot

```diff
@@ -0,0 +1,144 @@
+*** Settings ***
+Documentation       Tests related to greenboot
+
+Resource            ../resources/common.resource
+Resource            ../resources/systemd.resource
+Resource            ../resources/ostree-health.resource
+Resource            ../resources/ostree.resource
+Resource            ../resources/microshift-config.resource
+
+Suite Setup         Setup Suite
+Suite Teardown      Teardown Suite
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${LOCAL_WORKLOAD_FILE}          ../docs/config/busybox_running_check.sh
+${GREENBOOT_WORKLOAD_FILE}      /etc/greenboot/check/required.d/50_busybox_running_check.sh
+${GREENBOOT_CONFIG_FILE}        /etc/greenboot/greenboot.conf
+${GREENBOOT_CONFIG_CONTENT}     MICROSHIFT_WAIT_TIMEOUT_SEC=180
+${MANIFEST_SOURCE_DIR}          ./assets/kustomizations/greenboot/
+${MANIFEST_DIR}                 /etc/microshift/manifests
+${HOSTNAME_BIN_PATH}            ""
+
+
+*** Test Cases ***
+GreenbootHealth Run with User Workload
```

Comment:
Do we need `GreenbootHealth`? All these tests will be grouped by the `greenboot.robot` test suite, so I think we might drop the prefix